jme3-android-native/src/native/jme_decode/com_jme3_audio_plugins_NativeVorbisFile.c

```diff
@@ -171,7 +176,7 @@ JNIEXPORT void JNICALL Java_com_jme3_audio_plugins_NativeVorbisFile_nativeInit
     nvf_field_duration = (*env)->GetFieldID(env, clazz, "duration", "F");
 }
 
-JNIEXPORT void JNICALL Java_com_jme3_audio_plugins_NativeVorbisFile_open
+JNIEXPORT void JNICALL Java_com_jme3_audio_plugins_NativeVorbisFile_init
   (JNIEnv *env, jobject nvf, jint fd, jlong off, jlong len)
 {
     LOGI("open: fd = %d, off = %lld, len = %lld", fd, off, len);
```

Comment:
Should this be `LOGI("init:...`

Reply:
Yes, as i mentioned above, i will refactor old native logs soon..